.github/workflows/deploy.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+   workflow_run:
+    workflows: ["Test"]
+    branches: [main]
+    types: 
+      - completed
+
+jobs:
+  deploy:
+    name: Deploy to cloud.gov
+    needs: build_and_test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```

Comment:
It doesn't really need this line anymore.

Reply:
👍  removed this. @ondrae please merge if it works for you!